components/scream/CMakeLists.txt

```diff
@@ -42,7 +42,8 @@ if (Kokkos_ENABLE_CUDA)
   EkatSetNvccWrapper()
 endif()
 
-set (CMAKE_CXX_STANDARD 11)
+#set (CMAKE_CXX_STANDARD 11)
```

Comment:
Is this an issue? @bartgol , are we cool with moving to C++14?

Reply:
FYI, I believe this was to work around something in YAKL. I can try downgrading, Matt has since upgraded YAKL.

Reply:
Just checked this, I can remove this dependence and we can decide independently of this PR whether or not to move to C++14.

Reply:
Removed.